files/pt-br/web/svg/element/use/index.html

```diff
@@ -48,11 +48,11 @@ <h2 id="Atributos">Atributos</h2>
 <h3 id="Atributos_globais">Atributos globais</h3>
 
 <ul>
- <li><a href="/pt-BR/SVG/Attribute#ConditionalProccessing" title="en/SVG/Attribute#ConditionalProccessing">Atributos de processamento condicional</a> »</li>
- <li><a href="/pt-BR/SVG/Attribute#Core" title="en/SVG/Attribute#Core">Atributos principais</a> »</li>
- <li><a href="/pt-BR/SVG/Attribute#GraphicalEvent" title="en/SVG/Attribute#GraphicalEvent">Atributos de eventos gráficos</a> »</li>
- <li><a href="/pt-BR/SVG/Attribute#Presentation" title="en/SVG/Attribute#Presentation">Atributos de apresentação</a> »</li>
- <li><a href="/pt-BR/SVG/Attribute#XLink" title="en/SVG/Attribute#XLink">Atributos XLink</a> »</li>
+ <li><a href="/pt-BR/Web/SVG/Attribute#ConditionalProccessing" title="en/SVG/Attribute#ConditionalProccessing">Atributos de processamento condicional</a> »</li>
+ <li><a href="/pt-BR/Web/SVG/Attribute#Core" title="en/SVG/Attribute#Core">Atributos principais</a> »</li>
+ <li><a href="/pt-BR/Web/SVG/Attribute#GraphicalEvent" title="en/SVG/Attribute#GraphicalEvent">Atributos de eventos gráficos</a> »</li>
+ <li><a href="/pt-BR/Web/SVG/Attribute#Presentation" title="en/SVG/Attribute#Presentation">Atributos de apresentação</a> »</li>
+ <li><a href="/pt-BR/Web/SVG/Attribute#XLink" title="en/SVG/Attribute#XLink">Atributos XLink</a> »</li>
```

Comment:
Obrigado pela contribuição @davipguimaraes!!

Olhando o original aqui (https://raw.githubusercontent.com/mdn/content/main/files/en-us/web/svg/element/use/index.md) parece que faltou também um `/web`.

Fiz a sugestão no seu PR.. O que acha?

```suggestion
 <li><a href="/pt-BR/docs/Web/SVG/Attribute#ConditionalProccessing" title="en/SVG/Attribute#ConditionalProccessing">Atributos de processamento condicional</a> »</li>
 <li><a href="/pt-BR/docs/Web/SVG/Attribute#Core" title="en/SVG/Attribute#Core">Atributos principais</a> »</li>
 <li><a href="/pt-BR/docs/Web/SVG/Attribute#GraphicalEvent" title="en/SVG/Attribute#GraphicalEvent">Atributos de eventos gráficos</a> »</li>
 <li><a href="/pt-BR/docs/Web/SVG/Attribute#Presentation" title="en/SVG/Attribute#Presentation">Atributos de apresentação</a> »</li>
 <li><a href="/pt-BR/docs/Web/SVG/Attribute#XLink" title="en/SVG/Attribute#XLink">Atributos XLink</a> »</li>
```

Reply:
@[josielrocha](https://github.com/josielrocha) eu não hávia notado o link com o `/docs`, mas acrescentei na correção.